libs/designsystem/src/lib/components/progress-circle/progress-circle-ring.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, Input, ChangeDetectionStrategy, HostBinding } from '@angular/core';
+
+import { ThemeColor } from '../../helpers/theme-color.type';
+
+const STROKE_WIDTH = 4;
```

Comment:
This is to avoid it being af member variable. For actual constants i would argue they should be placed there. Do you think it should be a member variable?

Reply:
Saw other comment about making it readonly. Good suggestion. Fixed.